plugins/appstatus.py

```diff
@@ -0,0 +1,81 @@
+import subprocess
+import sys
+import json
+from crmetrics import CRBase
+
+
+class AppStatusFinder(CRBase):
+
+    def get_app_instance_status(self, kind, instance, kubeconfig):
+        cmd = 'kubectl get %s %s -o json %s' % (kind, instance, kubeconfig)
+        out, err = self.run_command(cmd)
+        if err != "":
+            print("Something went wrong while getting app instance status.")
+            print(err)
+            exit(1)
+
+        response = json.loads(out)
+        if 'status' in response:
+            if 'helmrelease' in response['status']:
+                helm_release = response['status']['helmrelease'].strip('\n')
+                ns, name = helm_release.split(':')
+                return name, ns, True
```

Comment:
Can you use a variable name for the last parameter as well?

Reply:
Done!